test/e2e/elasticache/tests/test_replicationgroup_largecluster.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Large cluster load test for replication group resource
+"""
+
+import pytest
+
+from time import sleep
+from common import k8s
+from elasticache.tests.test_replicationgroup import make_replication_group, rg_deletion_waiter, make_rg_name, DEFAULT_WAIT_SECS
```

Comment:
you import DEFAULT_WAIT_SECS from test_replicationgroup but override this variable name below on line 24 :)

Reply:
will delete the override, I was playing around with different times and forgot I had imported it after I settled on 30